docs/client_quickstart.rst

```diff
@@ -187,7 +187,8 @@ The ``gzip`` and ``deflate`` transfer-encodings are automatically
 decoded for you.
 
 You can enable ``brotli`` transfer-encodings support,
-just install  `Brotli <https://pypi.org/project/Brotli>`_.
+just install `brotlicffi <https://pypi.org/project/brotlicffi/>`_
+or `Brotli <https://pypi.org/project/Brotli/>`_.
```

Comment:
I think brotli should be listed first as it's what most people will want.
Though, maybe this section should be updated to just recommend installing `aiohttp[brotli]` instead, then users don't need to worry about that.

Reply:
I've reversed the order for now, as I suppose bigger changes in the docs deserve a PR of their own.